### DISCUSSION TITLE
DAI 2023-2024 - Practical work 1 - Alexandre Iorio

### DISCUSSION THREAD

**Discussion by AlexandreIorio** [heig-vd-dai-course/heig-vd-dai-course#79] [Show and tell]
I'm going to create a program that can overlay text on an image. It will be able to retrieve the EXIF data from the images if they are present. The output files will be in JPG or PNG format, depending on the user's preference. This program can be used as a PNG to JPG converter (and vice versa).

**Comment by AlexandreIorio** [comment]
here is the link to the project : [https://github.com/AlexandreIorio/Inko](https://github.com/AlexandreIorio/Inko)

**Comment by ludelafo** [comment]
Looks great! Out of curiosity, did you accomplish this all by yourself in just two days? Did you have a source inspiration?

**Comment by AlexandreIorio** [comment]
It's a long story of unverified commits and wrong pushes. 

I started the project in the lab last week and have been working on it a lot (too much), and it's really inspired me. Due to problems with git, I recreated each commit by downloading the commit ZIP and committing it to the INKO project with the same name. I've understood my mistake with the unverified commits (for the second time) and I won't do it again. 

I was inspired by the automation process of the XNView program, which can write to image files. [https://www.xnview.com/fr/](https://www.xnview.com/fr/)
